config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```diff
@@ -117,7 +117,7 @@ periodics:
         - name: ETCD_VERSION_UPGRADE_TO
           value: "3.4.13-0"
         - name: COREDNS_VERSION_UPGRADE_TO
-          value: "1.8.0"
+          value: "v1.8.0"
```

Comment:
Let's fix 1.7.0 above for consistency (it is not a problem because the version is not changing)

Reply:
I think it will break because kubeadm will concatenate the version with the old prefix => `k8s.gcr.io/coredns:v1.7.0`

This image unfortunately does not exist

Reply:
for kubeadm 1.21:
https://github.com/kubernetes/kubernetes/blob/release-1.21/cmd/kubeadm/app/constants/constants.go#L330-L334

1.20:
https://github.com/kubernetes/kubernetes/blob/release-1.20/cmd/kubeadm/app/constants/constants.go#L332
https://github.com/kubernetes/kubernetes/blob/release-1.20/cmd/kubeadm/app/constants/constants.go#L353